<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newEntry</class>
 <widget class="QMainWindow" name="newEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addBook">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>FIlm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTVEpisode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TV Episode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVideoGame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select catagory to add</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
